include/velocypack/Slice.h

```diff
@@ -181,9 +219,14 @@ class Slice {
     return VELOCYPACK_HASH(start(), static_cast<std::size_t>(stringSliceLength()), seed);
   }
 
-  // check if slice is of the specified type
+  // check if slice is of the specified type (excluding tags)
   constexpr inline bool isType(ValueType t) const noexcept {
-    return SliceStaticData::TypeMap[*_start] == t;
+    return SliceStaticData::TypeMap[*start()] == t;
```

Comment:
IMHO this should be reverted to just return the type based on `_start` again, as this will ensure the old behavior that embedders may rely on.
```suggestion
    return SliceStaticData::TypeMap[_start] == t;
```